src/main/webapp/portal/main/partials/server-error.html

```diff
@@ -0,0 +1,13 @@
+<frame-page app-header-title='{{NAMES.title}} - Service Temporarily Unavailable' 
+            app-header-collapse='true'
+            app-header-image='img/terrace.jpg'>
+  <h2 class='center'>We're sorry, but there was a problem with your request.</h2>
+  <p class='center'>
+    <i class='fa fa-exclamation-triangle fa-3x' style='color: #b70101;'></i>
```

Comment:
Hardcoded color?  Could this be an alias?  Do we need CSS classes for error messaging?